content/_data/events/2018-12-05-philadelphia-jam.adoc

```diff
@@ -0,0 +1,6 @@
+---
+title: "Philadelphia JAM"
+date: “2018-12-05T18:00:00” 
+link: "https://www.meetup.com/Philadelphia-Jenkins-Area-Meetup"
+---
+Philadelphia JAM
```

Comment:
It's not much helpful, it would be preferable to have a summary instead

Reply:
There isn't any more information on the site. This is the best that can be done right now.